client/apps/edit/components/header/index.coffee

```diff
@@ -11,15 +11,28 @@ module.exports = class EditHeader extends Backbone.View
     @user = new User sd.USER
     @article.on 'change', @saving
     @article.on 'change', @toggleCheckmarks
+    @article.on 'error', @errorSaving
     @article.on 'sync', @doneSaving
     @toggleCheckmarks()
 
+  errorSaving: (model, e) =>
+    message = JSON.parse(e.responseText).message ? e.responseText
+    console.error "Error saving #{model.get('id')}: #{e.status}: #{message}."
+    @$('#edit-save')
+      .text('Error')
+      .attr('title', message)
```

Comment:
You can hover over the button and see the error.

![eror](https://cloud.githubusercontent.com/assets/542335/24764135/68a7d4c4-1ac1-11e7-8305-416ca33f1216.gif)